### ISSUE TITLE
Gif support?

### ISSUE THREAD

**Issue by JustinBacher** [sxyazi/yazi#1195]
### Please describe the problem you're trying to solve

I see on that this has [not yet been requested](https://github.com/sxyazi/yazi/issues/51). Is this planned? Is this something you're hoping someone would implement in a PR?

I could see about getting this working although I don't have a clue where to start if I could get a nudge in the right direction.

### Would you be willing to contribute this feature?

- [X] Yes, I'll give it a shot

### Describe the solution you'd like

Be able to have gif's/movies/animated images work in previewer

**Comment by sxyazi** [comment]
Yeah I would like to give it a try sometime in the future to see if there's an efficient way to achieve it, it's not a priority for me, but I always welcome any constructive PRs.

Displaying GIFs in the terminal isn't an easy task; we need to consider support for different protocols (AFAIK, kitty and iTerm2 support GIFs, but Sixel doesn't, so we might need to manually split frames for images). There could also be performance problems with how to preload GIF files.

**Comment by sxyazi** [comment]
Tracked in https://github.com/sxyazi/yazi/issues/51, closing in favor of that list
